### ISSUE TITLE
Score embedding clusters wrt user labels and then gesture labels

### ISSUE THREAD

**Issue by kdmalc** [kdmalc/fl-gestures#20]

**Comment by kdmalc** [comment]
And also maybe the xlsx labels?

**Comment by kdmalc** [comment]
And also test the Autoencoder embeddings too? After you do PCA for this

**Comment by kdmalc** [comment]
https://scikit-learn.org/stable/api/sklearn.metrics.html

All of the following compare to sets of clustering labels. For the "ground_truth"\labels_true, iterate through the different xlsx labels (User ID, Gesture ID, hand used, muscles activated 1)

Adjusted Mutual Information Score, Adjusted Rand Score, Completeness Score, Contingency Matrix, Pair Confusion Matrix, Fowlkes-Mallows Score, Homogeneity, Completeness, and V-Measure Scores, Homogeneity Score, Mutual Information Score, Normalized Mutual Information Score, Rand Score

**Comment by kdmalc** [comment]
Irrelevant to current project goals.
